srcpkgs/diffuse/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'diffuse'
 pkgname=diffuse
-version=0.8.1
+version=0.8.2
 revision=1
 build_style=meson
 hostmakedepends="gettext"
```

Comment:
```suggestion
hostmakedepends="gettext pkg-config"
```

Reply:
Thank you. The arm builds are still failing because of a missing `glib-compile-resources`; I'm not sure what else to try since `glib-devel` is already in the makedepends

Reply:
add glib-devel to hostmakedepends too

Reply:
Many thanks, I should've tried that first. I'm still pretty new to building/troubleshooting source packages